### ISSUE TITLE
create a Yeoman based project wizard UI so folks can spin up nodejs projects easily from reusable templates

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8#5505]

**Comment by rawlingsj** [comment]
Yeah I think this could make node feel more like a first class citizen in fabric8

**Comment by jstrachan** [comment]
I guess we could also just try create some archetypes for nodejs / golang projects; which would mean we could reuse the same tooling ;) but a pure nodejs / yeoman tool would be cooler; letting folks add various generators from yeoman
